README.md

```diff
@@ -68,6 +69,24 @@ This software package includes solvers for continuous optimization problems. The
     <img src="ccvm_simulators/ccvmplotlib/images/success_prob_plot_example.png" width="250">
 </p>
 
+
+## Architecture and Workflow Overview
+#### Example Workflow
+The workflow diagram showcases, step-by-step, how the CCVM simulators can be
+used to solve a problem.
```

Comment:
We should mention here about the two different "END" states you can get to depending if you want to plot the solution or just print the results

Reply:
Sure. Done.